components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -394,19 +546,18 @@ void RRTMGPRadiation::initialize_impl(const RunType /* run_type */) {
   m_do_subcol_sampling = m_params.get<bool>("do_subcol_sampling",true);
 
   // Initialize yakl
-  yakl_init();
+  init_kls();
```

Comment:
What is `init_kls`?

Reply:
initializes all kernel launchers, yakl and kokkos.